robomimic/envs/env_robosuite.py

```diff
@@ -127,13 +132,19 @@ def step(self, action):
         obs = self.get_observation(obs)
         return obs, r, self.is_done(), info
 
-    def reset(self):
+    def reset(self, unset_ep_meta=True):
         """
         Reset environment.
 
+        Args:
+            unset_ep_meta (np.array): whether to reset any previously set episode meta data
+        
         Returns:
             observation (dict): initial observation dictionary.
         """
+        if unset_ep_meta and hasattr(self.env, "unset_ep_meta"):
```

Comment:
is there any flag we can set on env construction that detects if the env is a robocasa env, so we don't need to keep checking env attributes? or do you anticipate this being a property that will be integrated back into robosuite / other places?

Reply:
it's already a feature that's integrated into robosuite v1.5, but it's not really being used (it's mostly a no-op there), However in the future it may be used for non-robocasa envs too. I would try to keep this general logic for future-proofing. It's backwards compatible.